src/plugins/intel_gpu/src/kernel_selector/kernels/reduce/reduce_kernel_b_fs_yx_fsv16.cpp

```diff
@@ -216,13 +216,14 @@ JitConstants ReduceKernel_b_fs_yx_fsv16::GetJitConstants(const reduce_params& pa
         }
     }
 
-    // MIN/MAX mode should handle feature remainder in case reduce axes includes feature
-    if (params.reduceMode == ReduceMode::MIN || params.reduceMode == ReduceMode::MAX) {
-        if (count(params.reduceAxes.begin(), params.reduceAxes.end(), 1) > 0) {
-            if (params.inputs[0].Feature().v % 16 != 0) {
-                jit.AddConstant(MakeJitConstant("HANDLE_FEATURE_REMAINDER", 1));
-            }
-        }
+    bool zero_invariant_mode = params.reduceMode == ReduceMode::L1 || params.reduceMode == ReduceMode::L2 ||
+                               params.reduceMode == ReduceMode::LOG_SUM ||
+                               params.reduceMode == ReduceMode::LOG_SUM_EXP || params.reduceMode == ReduceMode::MEAN ||
+                               params.reduceMode == ReduceMode::OR || params.reduceMode == ReduceMode::SUM ||
+                               params.reduceMode == ReduceMode::SUM_SQUARE;
+    if (zero_invariant_mode || params.inputs[0].Feature().v % 16 == 0) {
+        if (count(params.reduceAxes.begin(), params.reduceAxes.end(), 1) > 1)
```

Comment:
I think this should be         `if (count(params.reduceAxes.begin(), params.reduceAxes.end(), 1) == 1)`. isn't it?

Reply:
Sorry for the typo. `count(axes, 1) > 0` is correct because kernel only use `ZERO_INVARIANT_MODE` only when it has feature_axis(= axis 1) reduce.